WordPress/src/main/java/org/wordpress/android/ui/news/NewsManager.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.android.ui.news
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MediatorLiveData
+import org.wordpress.android.models.news.NewsItem
+import org.wordpress.android.ui.prefs.AppPrefsWrapper
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Business logic related to fetching/showing the News Card (a card used for announcing new features/updates).
+ */
+@Singleton
+class NewsManager @Inject constructor(val newsService: NewsService, val appPrefs: AppPrefsWrapper) {
+    private val dataSourceMediator: MediatorLiveData<NewsItem?> = MediatorLiveData()
+    private var localServiceDataSource: LiveData<NewsItem?>? = null
```

Comment:
I think you can initialise the source right here instead of doing it lazily in the `newsItemSource`. You also don't really need to care if it's local or not. That's hidden behind the interface so I'd rename it to maybe `newsItemDataSource`?

Reply:
As discussed - I'd say the splitting the two - initialization and pulling - is what we should do. Pull can be conditional (do we have data? have we pulled already?).